jetson/percep/main.cpp

```diff
@@ -146,11 +146,9 @@ int main() {
 
      //Update LCM 
     obstacleMessage.bearing = lastObstacle.bearing; //update LCM bearing field
-    if(lastObstacle.distance <= obstacle_detection.distance)
-      obstacleMessage.distance = (lastObstacle.distance/1000); //update LCM distance field
-    else
-      obstacleMessage.distance = (obstacle_detection.distance/1000); //update LCM distance field
-    cerr << "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!Path Sent: " << obstacleMessage.bearing << "\n";
+    obstacleMessage.distance = lastObstacle.distance; //update LCM distance field
+    cerr << "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!Path Bearing Sent: " << obstacleMessage.bearing << "\n";
+    cerr << "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!Distance Sent: " << obstacleMessage.distance << "\n";
```

Comment:
Is there a reason for the formatting of this error message? It seems excessive

Reply:
Since we currently have a lot of output it's very helpful to have the offset for debugging purposes. We are currently working on cleaning up our debug output in Issue #460

Reply:
Are we leaving cerrs as is? Or should we start moving towards couts?

Reply:
we are moving towards cout. Don't have to worry ab fixing that in this PR